e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.Registry.cs

```diff
@@ -234,17 +234,10 @@ public static class Registry {
                 request.AddHeader(TestConstants.HttpHeaderNames.Authorization, accessToken);
                 request.Resource = string.Format(TestConstants.APIRoutes.RegistryActivateEndpointsFormat, endpointId);
 
-                // This request fails when called for the first time, retry the call while the bug is being investigated
```

Comment:
your fix is for this workaround?

Reply:
exactly. the issue was that the first activate endpoint always fails